create_framework/http_kernel_httpkernelinterface.rst

```diff
@@ -46,8 +46,8 @@ Update your framework so that it implements this interface::
         }
     }
 
-Even if this change looks not too complex, it brings us a lot! Let's talk about one of
-the most impressive one: transparent :doc:`HTTP caching </http_cache>` support.
+Even if this change doesn't look like much, it brings us a lot! Let's talk about one of
```

Comment:
Can we rephrase this differently? 

Not to say that any change is "small" or "simple".

Reply:
Maybe "Even if this change looks minor..." then?

Reply:
> Or "With this change, a little goes a long way !"

Perfect!

Context for other reviewers: The change is to add an interface.

Reply:
Then I would go with:
"With this change, a little goes a long way! Let's talk about one of the most impressive upsides:"

Reply:
Sounds. good. Thank you.